include/roaring/bitset_util.h

```diff
@@ -569,6 +572,282 @@ CROARING_TARGET_AVX2
 AVXPOPCNTFNC(andnot, _mm256_andnot_si256)
 CROARING_UNTARGET_REGION
 
```

Comment:
Why don't we go straight to requiring VPOPCNTDQ which both Ice Lake and Zen 4 support. Then we can simply do `_mm512_popcnt_epi64`.

Reply:
I agree. how about implement in the next release? so this release can apply servers not only icelake but also casalake...